examples/run.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2020 Fredy Ramirez <https://formateli.com>
+# For license see LICENSE
+"""
+run.py
+Show all examples located in this example folder.
+Main window show a button for each example.
+"""
+import sys, os
+import subprocess
+import tkinter as tk
+from tkinter import ttk
+
+_DIR = os.path.dirname(os.path.realpath(__file__))
+_DIR = os.path.normpath(os.path.join(_DIR, '..'))
+
+EXAMPLE_ENV = os.environ.copy()
+if not 'PYTHONPATH' in EXAMPLE_ENV:
+    EXAMPLE_ENV["PYTHONPATH"] = _DIR
+else:
+    EXAMPLE_ENV["PYTHONPATH"] = _DIR + os.pathsep + EXAMPLE_ENV["PYTHONPATH"]
+
+
+class _SampleButton:
+    def __init__(self, root, text, col, row):
+        self.root = root
+        self.btn = ttk.Button(root, text=text, command=self.run_example)
+        self.btn.grid(row=row, column=col, sticky="nsew") 
+
+    def run_example(self, event=None):
+        try:
+            subprocess.run(
+                [
+                    'python',
+                    'example_' + self.btn['text'] + '.py'
+                ],
+                env=EXAMPLE_ENV
+            )
+        except Exception as e:
+            print(e)
+
+
+def _get_samples():
+    result = []
+    dir_ = os.path.dirname(os.path.realpath(__file__))
+    files = os.listdir(dir_)
+    for f in files:
+        if f == 'run.py':
+            continue
+        fp = os.path.join(dir_, f)
+        if os.path.isfile(f) and f.endswith('.py'):
+            f = f[8:] # remove example_
```

Comment:
Please use two spaces before a line comment, also for [PEP-8 compliance](https://www.python.org/dev/peps/pep-0008/#inline-comments).

Reply:
Done